esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -148,6 +151,12 @@ class esp_http_decl CSoapRequestBinding : public CSoapComplexType,
     void setThunkHandle(void * val){thunk_=val;}
     void * getThunkHandle(){return thunk_;}
 
+    void setConnectTimeOutMs(unsigned timeout){connectTimeoutMs = timeout;}
```

Comment:
@rpastrana Are you going to suggest that (or maybe someone already has?) as a new coding standard for the implementation of all inherited interface methods?